### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by kennysong**
The documentation wasn't reflecting the latest German and Chinese changes. (We need to run the `sphinx-apidoc` command manually when the directory structure changes.)

To be opened after https://github.com/citadel-ai/langcheck/pull/97 is merged.

### CODE REVIEW

**Comment by kennysong** [comment]
@liwii Can you review this too? It's a minor documentation change.

File: docs/contributing.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ To run tests:
 To make documentation:
 
 1. **Optional:** Re-generate all `docs/langcheck*.rst` files.
-   - `sphinx-apidoc -f --no-toc --separate --module-first -t docs/_templates/ -o docs src/langcheck/ src/langcheck/stats.py`
+   - `sphinx-apidoc -f --no-toc --separate --module-first -t docs/_templates/ -o docs src/langcheck/ src/langcheck/stats.py src/langcheck/metrics/model_manager`
```
Comment (liwii) [inline docs/contributing.md:48]: Do we need change? I tried running both command in my env but the outputs seem to be the same

Comment (liwii) [reply docs/contributing.md:48]: Related: don't we need to add `docs/langcheck.metrics.model_manager.rst`?

**Review by liwii — COMMENTED** [review]
Mostly LGTM, left a few minor comments

File: docs/contributing.md (hunk 1 above)
Comment (kennysong) [reply docs/contributing.md:48]: The page is totally empty if we include it, so I intentionally excluded it.

CleanShot 2024-04-02 at 19 24 51@2x

**Review by liwii — APPROVED** [review]
LGTM after resolving the hanlp tokenizer errors

**Comment by kennysong** [comment]
Thanks! The HanLP error is because https://file.hankcs.com/hanlp/tok/fine_electra_small_20220615_231803.zip is down again. If it persists for more than a week we can look into options to fix it.

**Pull request merged**